wa/framework/configuration/core.py

```diff
@@ -891,16 +891,16 @@ class JobSpec(Configuration):
                            Similar to IDs but do not have the uniqueness restriction.
                            If specified, labels will be used by some result
                            processes instead of (or in addition to) the workload
-                           name. For example, the csv result processor will put
+                           name. For example, the csv output processor will put
                            the label in the "workload" column of the CSV file.
                            '''),
         ConfigurationPoint('augmentations', kind=toggle_set, merge=True,
                            aliases=["instruments", "processors", "instrumentation",
-                                    "result_processors", "augment"],
+                                    "output_processors", "augment"],
                            description='''
-                           The instruments and result processors to enable (or
+                           The instruments and output processors to enable (or
                            disabled using a ~) during this workload spec. This combines the
-                           "instrumentation" and "result_processors" from
+                           "instrumentation" and "output_processors" from
```

Comment:
Here, it should actually keep saying `"result_processors"` as this talks about the names of these configs in WA2.

Reply:
Fixed